interface/ceed-operator.c

```diff
@@ -503,6 +504,195 @@ int CeedOperatorAssembleLinearDiagonal(CeedOperator op, CeedVector *assembled,
   return 0;
 }
 
+/**
+  @brief Build a FDM based approximate inverse for each element for a
+           CeedOperator
+
+  This returns a CeedOperator and CeedVector to apply a Fast Diagonalization
+    Method based approximate inverse. This function obtains the simultaneous
+    diagonalization for the 1D mass and Laplacian operators,
+      M = V^T V, K = V^T S V.
+    The assembled QFunction is used to modify the eigenvalues from simultaneous
+    diagonalization and obtain an approximate inverse of the form
+      V^T S^hat V.
```

Comment:
What is assumed about the input CeedOperator? Does the QFunction matter? Can it be composite? I assume there are a few assumptions.

Reply:
Right, for now it's non-composite operators with linear qfunctions. I'll document that. I plan on relaxing that in the future.